types/n3/n3-tests.ts

```diff
@@ -152,6 +152,22 @@ function test_doc_rdf_stream_to_triples_1() {
     }());
 }
 
+function test_doc_rdf_stream_to_triples_2() {
+    interface QuadBnode extends N3.BaseQuad {
+        subject: N3.BlankNode;
+        predicate: N3.BlankNode;
+        object: N3.BlankNode;
+        graph: N3.BlankNode;
+    }
```

Comment:
What is the reason for definining this type?

Reply:
@mielvds ?

Reply:
I don't know, it was borrowed from a previous test: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/n3/n3-tests.ts#L131

Reply:
I don't know, it was borrowed from a previous test: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/n3/n3-tests.ts#L131